common/utils.js

```diff
@@ -385,6 +385,7 @@
             var contextObj = ConfigUtils.getContextJSON();
             pcontext.push("pcid=" + contextObj.cid);
             pcontext.push("ppid=" + contextObj.pid);
+            if (contextObj.hideNavbar != undefined) pcontext.push("hideNavbar=" + contextObj.hideNavbar)
```

Comment:
This is not very important but just to be consistent with record and recordset, you might want to add `hideNavbar` only when its value is `true`.